src-docs/src/views/range/range_example.js

```diff
@@ -89,6 +178,171 @@ export const RangeControlExample = {
+            the range handles will show at the min (or max and min) positions.
+          </p>
+        </Fragment>
+      ),
+      source: [{
+        type: GuideSectionTypes.JS,
+        code: inputSource,
+      }, {
+        type: GuideSectionTypes.HTML,
+        code: inputHtml,
+      }],
+      demo: <InputExample />,
+      props: { EuiRangeInput },
+      snippet: `<EuiRange showInput />
+
+<EuiDualRange showInput />`,
+    },
+    {
+      title: 'Tick marks',
+      text: (
+        <Fragment>
+          <p>
+            To show clickable tick marks and labels at a given interval, add the prop <EuiCode>showTicks</EuiCode>.
+            By default, tick mark interval is bound to the <EuiCode>step</EuiCode> prop, however, you can set a custom
+            interval without changing the actual steps allowed by passing a number to the <EuiCode>tickInterval</EuiCode> prop.
+          </p>
+          <p>
+            To pass completely custom tick marks, you can pass an array of objects that
+            require a <EuiCode>value</EuiCode> and <EuiCode>label</EuiCode>. The value must be included in the range of values
+            (min-max), though the label may be anythin you choose.
+          </p>
+          <EuiCallOut color="warning" title="Maximum of 20 ticks allowed">
+            <p>Spacing can get quite cramped with lots of ticks so we max out the number to 20.</p>
+          </EuiCallOut>
+        </Fragment>
+      ),
+      source: [{
+        type: GuideSectionTypes.JS,
+        code: ticksSource,
+      }, {
+        type: GuideSectionTypes.HTML,
+        code: ticksHtml,
+      }],
+      demo: <TicksExample />,
+      props: { EuiRangeTicks },
+      snippet: `<EuiRange step={10} showTicks />
+
+<EuiRange showTicks tickInterval={20} />
+
+<EuiDualRange
+  showTicks
+  ticks={[
+    { label: '20kb', value: 20 },
+    { label: '100kb', value: 100 }
+  ]}
+/>`,
+    },
+    {
+      title: 'Levels',
+      text: (
+        <Fragment>
+          <p>
+            To create colored indicators for certain intervals, pass an array of objects that
+            include a <EuiCode>min</EuiCode>, <EuiCode>max</EuiCode> and <EuiCode>color</EuiCode>.
+            Color options are <EuiCode>{JSON.stringify(LEVEL_COLORS, null, 2)}</EuiCode>.
+          </p>
+          <p>
+            Be sure to then add an <EuiCode>aria-describedby</EuiCode> and match it to the
+            id of a <EuiCode>EuiFormHelpText</EuiCode>.
+          </p>
+        </Fragment>
+      ),
+      source: [{
+        type: GuideSectionTypes.JS,
+        code: levelsSource,
+      }, {
+        type: GuideSectionTypes.HTML,
+        code: levelsHtml,
+      }],
+      demo: <LevelsExample />,
+      props: { EuiRangeLevels },
+      snippet: `<EuiRange
+  levels={[
+    {min: 0, max: 20, color: 'danger'},
+    {min: 20, max: 100, color: 'success'}
+  ]}
+  aria-describedby=""
```

Comment:
Small docs comment. As snippets get more pervasive they will likely be the first entry point for peeps. Rather than leave blank values i'd put pseudo code in here to explain what to put in there. I know you have this written above in the example text, but it doesn't hurt to force the issue. Otherwise I bet people copying this might just clear it ("what is this, i don't really need it").

```suggestion
  aria-describedby="Put the id of the helper text or form label here"
```

Reply:
That's because I worry more that people will copy/paste, **not** replace that string and therefore leaving the aria label useless. At least with a blank string, it won't **hurt** accessibility.

Reply:
Talked with engineers and decided to go with a variable.